3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs

```diff
@@ -9,43 +9,17 @@ namespace SeleniumTest.Tests;
 public class UpdateFullNameStepDefinition : CommonSteps
 {
     private readonly HomePage _homePage;
-    private ProfilePage? _profilePage;
     private readonly ScenarioContext _scenarioContext;
     public UpdateFullNameStepDefinition(ScenarioContext scenarioContext) : base(scenarioContext)
     {
         _homePage = new HomePage();
         _scenarioContext = scenarioContext;
     }
 
-    [Given(@"the user clicks on the Settings option on the Nav Bar")]
-    public void GiventheuserclicksontheSettingsoptionontheNavBar()
+    [Then(@"the '(.*)' is updated")]
```

Comment:
This step could be a more general step to validate any text field element

Reply:
I thought of that but in this particular case I didn't leave it as a common step because it has to click on the Settings option from the Home Page inside the test.